plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3select/TrinoS3ClientFactory.java

```diff
@@ -15,14 +15,20 @@
 
 import com.amazonaws.ClientConfiguration;
 import com.amazonaws.Protocol;
+import com.amazonaws.SdkClientException;
```

Comment:
Can you please rebase the code? There have been some changes in the `TrinoS3FileSystem` in the meantime.